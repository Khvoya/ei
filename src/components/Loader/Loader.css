.Loader__wrap {
    width: 100vw;
    height: 100vh;
    min-height: 700px;
    background-color: black;
    z-index: 100;
    position: absolute;
    display: flex;
    justify-content: center;
    align-items: center;
}
.Loader__logo {
    width: 200px;
    height: 200px;
    border-radius: 50%;
    box-shadow: 0 0 10px 6px white;
    animation-name: spin;
    animation-duration: 3000ms;
    animation-iteration-count: infinite;
    animation-timing-function: ease-in;
}

@keyframes spin {
    from {
        transform:rotate(0deg);
    }
    to {
        transform:rotate(360deg);
    }
}
