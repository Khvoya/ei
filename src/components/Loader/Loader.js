import React from 'react';
import './Loader.css'
import logo from 'media/BY_OUTgnFUs.jpg';

const Loader = () => {
  return (
    <div className="Loader__wrap">
      <img className="Loader__logo" src={logo} alt="logo"/>
    </div>
  )
};

export default Loader;
