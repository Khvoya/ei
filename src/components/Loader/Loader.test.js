import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Loader from './Loader';

describe('Loader', () => {

  it('should render correctly', () => {
    const output = shallow(
      <Loader />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
