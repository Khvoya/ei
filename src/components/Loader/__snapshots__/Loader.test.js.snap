// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render correctly 1`] = `
<div
  className="Loader__wrap"
>
  <img
    alt="logo"
    className="Loader__logo"
    src="BY_OUTgnFUs.jpg"
  />
</div>
`;
